client/src/pages/AgentDepartment/AgentDepartment.tsx

```diff
@@ -0,0 +1,37 @@
+import { useNavigate } from 'react-router-dom';
+import { useDepartmentsQuery } from '../../generated/graphql-types';
+import { PartialDepartment } from '../AgentDepartment/AgentDepartment.type';
+
+function AgentDepartment() {
```

Comment:
Could we find a better name for this ? DepartmentIndex ? departmentsList ? departmentsMenu ?

Reply:
Maybe, but I think it's good to leave Agent at the start of the file name so that Agent pages will be sorted in the same location in the codebase